app/Battle/great_battle.py

```diff
@@ -0,0 +1,14 @@
+from app.Knights.knight_class import Knight
```

Comment:
There is no need to use the word `great` in file naming, it does not contribute to a better understanding of its content. You could use `support_functions.py` or `battle_functions.py` here.

Reply:
👍